serverless/src/index.ts

```diff
@@ -55,6 +55,7 @@ export interface FunctionProperties {
   TracingConfig?: { [key: string]: string };
   FunctionName?: string;
   Architectures?: [string];
+  PackageType?: string;
```

Comment:
Is this property requested by AWS for Container based Lambdas?

Reply:
Just looked at the docs https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html, so it looks like yes.

Reply:
Correct, it's set to `Image` for Container based Lambdas.